package.json

```diff
@@ -51,6 +52,7 @@
     "babel-loader": "^6.2.0",
     "babel-plugin-add-module-exports": "^0.1.2",
     "babel-plugin-transform-dev-warning": "^0.1.0",
+    "babel-plugin-transform-object-assign": "git://github.com/newoga/babel-plugin-transform-object-assign.git",
```

Comment:
It would be good to publish it. If it's just designed to be used with material-ui and won't help the comminity, we can even consider moving the source code to this repository.